source/_docs/guides/drupal-8-commerce.md

```diff
@@ -0,0 +1,119 @@
+
+2.  This guide uses several variables in example [Terminus](/docs/terminus) commands. This lets you copy and paste without needing to change the variable. For this to work, you must first export the variables in your local terminal session:
+
+    ```bash
+    export SITENAME=yoursitenamehere
+    export GITHUB_TOKEN=yourgithubtokenhere
+    export CIRCLE_TOKEN=yourcirclecitokenhere
+    ```
+
+## Create A New Drupal 8 Site
+
+1.  Using the Terminus Build Tools plugin, create a new Drupal 8 site from the Pantheon [Example Drops 8 Composer](https://github.com/pantheon-systems/example-drops-8-composer) repository on GitHub:
+
+    ```bash
+    terminus build:project:create pantheon-systems/example-drops-8-composer $SITENAME
+    ```
+    At this point *do not* go to the web interface to continue installation.
+
+2.  Use Terminus to retrieve the Git URL for your new site and pass it to an environment variable:
+
+    ```
+    export GITURL=`terminus connection:info $SITENAME.dev --field=git_url`
+    ```
+
+2.  You now have a repository on GitHub containing your new site. Clone a local copy to your `projects` folder:
+
+    ```bash
+    cd ~/projects
+    git clone git@github.com:username/$SITENAME.git
+    ```
+
+    Remember to replace `username` with your GitHub username.
+
+## Install Drupal Commerce
+
+1. Move into the local repository for your site:
+
+        cd $SITENAME
+
+2. Use Composer to install the Commerce base:
+
+   ```bash
+   composer config repositories.commerce_base vcs https://github.com/drupalcommerce/commerce_base
+   composer require "drupalcommerce/commerce_base dev-8.x-1.x" "drupal/commerce:~2.0"  "drupal/admin_toolbar:~1.0"  "drupal/swiftmailer:~1.0"
+   ```
+
+    <div class="alert alert-info" role="alert">
+    <h4 class="info">Note</h4>
+    <p markdown="1">The commands above include dependencies required but not listed for `commerce_base`. You can follow the [open GitHub issue](https://github.com/drupalcommerce/commerce_base/pull/7){.external} for more information.</p>
+    </div>
+
+3. Running `git status` should show that the `composer.json` and `composer.lock` files have changed:
+
+   ![Git Status showing updated Composer files](/source/docs/assets/images/guides/drupal-8-commerce/git-status.png)
+
+4. Commit the new files and push them to GitHub:
+
+   ```bash
+   git commit -am "add commerce_base to project"
+   git push origin master
+   ```
+
+5. Go to your newly created Site Dashboard. Under the <span class="glyphicons glyphicons-wrench"></span>**Dev** tab, click on <span class="glyphicons glyphicons-embed-close"></span>**Code**, then **install later**. You should now see your commit history. Once CircleCI completes the automated tests built into our upstream repository, it will commit the build assets and push them to Dev:
```

Comment:
We may need to point people to the CircleCI output just because they'll be waiting a few minutes.

Also let's not use the word `upstream` at all in here.